packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -57,13 +58,21 @@ const DEFAULT_OPTIONS: PluginOptions = {
   rehypePlugins: [],
   showLastUpdateTime: false,
   showLastUpdateAuthor: false,
+  admonitions: {},
 };
 
 export default function pluginContentDocs(
   context: LoadContext,
   opts: Partial<PluginOptions>,
 ): Plugin<LoadedContent | null> {
   const options = {...DEFAULT_OPTIONS, ...opts};
+
+  if (opts.admonitions !== false) {
```

Comment:
Can just do `if (opt.admonitions) {`

Reply:
We cannot do this because `opts.admonitions` can be an object (even empty) or a boolean value (false).

Reply:
Why not? Empty objects are not falsey

Reply:
From docs:

> The admonitions key will be passed as the options to remark-admonitions. Passing false will prevent the plugin from being added to MDX.

I proceeded from this sentence, because the explicit value meaning better reflects our intention. Right?

Reply:
Ah, I get it your idea, I fixed all of them.